df1/lib/Df1/Render.hs

```diff
@@ -33,14 +38,14 @@ import Df1.Types
 
 --------------------------------------------------------------------------------
 
--- This is rather ugly, but whatever.
-renderColor :: Log -> BB.Builder
-{-# INLINABLE renderColor #-}
-renderColor = \log_ ->
- let t = renderIso8601 (log_time log_) <> space
+-- | Like 'log', but with ASCII colors.
```

Comment:
You meant “ANSI colors”, not “ASCII colors”.

While we are on it, perhaps rename this function to `logColorANSI`, to make it clear that these are ANSI color escapes, and not other kind of coloring.